### PR TITLE
Added new options to set range for address table and exception stack

### PR DESCRIPTION
**Pull request by Wlgen**
Signed-off-by: Ivan Diaz <ivan.diazortega@bsc.es>

### CODE REVIEW

**Comment by MikeOpenHWGroup** [comment]
Thanks for the contribution @Wlgen!

@noahsherrill, this PR needs to be reviewed and approved by you before it can be merged in.  AFAIK, you do not yet have "Committer" privileges with Eclipse, and so will not be able to execute the merge.  Once you approve this PR, I will merge it in.

**Review by noahsherrill — APPROVED** [review]
This looks good to me. Thank you, @Wlgen.

Resolves #44

**Pull request merged**
